src/Core/src/Core/IBorder.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Microsoft.Maui
+{
+	/// <summary>
+	/// Provides functionality to set the view borders.
+	/// </summary>
+	public interface IBorder
+	{
+		/// <summary>
+		/// Gets the the color of a view's border.
+		/// </summary>
+		Color BorderColor { get; }
```

Comment:
After merge the Brushes PR this must change to be a Brush and the property could be called `BorderBrush`.